hummingbot/strategy/hanging_orders_tracker.py

```diff
@@ -159,10 +164,15 @@ def is_hanging_order_in_strategy_active_orders(self, order: HangingOrder) -> boo
                        order.amount == o.quantity) for o in self.strategy.active_orders)
 
     def is_order_to_be_added_to_hanging_orders(self, order: LimitOrder) -> bool:
-        hanging_order = self._get_hanging_order_from_limit_order(order)
-        if hanging_order in self.equivalent_orders.union(self.orders_to_be_created):
-            return any(o.order_id == order.client_order_id
-                       for o in self.equivalent_orders.union(self.orders_to_be_created))
+        # Order will be added to hanging orders iff the corresponding order pair has been completely filled
```

Comment:
Currently `HangingOrderTracker` does not handle partially filled orders. We will be addressing this in another PR.